Makefile

```diff
@@ -63,13 +63,17 @@ dokka-javadoc:
 # Use `make generate-changelog TAG=LastReleaseTag` while running locally.
 .PHONY: generate-changelog
 generate-changelog:
-	mbx env; \
+	unset GITHUB_TOKEN; \
```

Comment:
Why do we need an unset?

Reply:
We login with file gh_token.txt, they will conflict and report error